pulsar-broker/src/test/java/org/apache/pulsar/broker/service/ReplicatedSubscriptionTest.java

```diff
@@ -1002,6 +1006,85 @@ public void testReplicatedSubscriptionWithCompaction() throws Exception {
         Assert.assertEquals(result, List.of("V2"));
     }
 
+    @Test
+    public void testReplicatedSubscriptionOneWay() throws Exception {
+        final String namespace = BrokerTestUtil.newUniqueName("pulsar-r4/replicatedsubscription");
+        final String topicName = "persistent://" + namespace + "/one-way";
+        config1.setReplicatedSubscriptionsSnapshotTimeoutSeconds(10);
```

Comment:
Is it possible to use a shorter timeout to make the test complete faster?

Reply:
Could the snapshot frequency be increased for the test as well?

Reply:
Yes, thanks for the suggestion. Addressed.